frontend/src/views/ContentView.svelte

```diff
@@ -0,0 +1,114 @@
+<style>
+  .content {
+    position: sticky;
+    top: 0;
+    height: 100%;
+    width: 100%;
+    overflow: hidden;
+  }
+
+  .breadcrumb {
+    padding-bottom: 1em;
+    background: var(--main-bg-color);
+  }
+
+  button {
+    margin-bottom: 0;
+    border: 1px solid white;
+  }
+
+  button:focus {
+    border-bottom: 2px solid var(--main-bg-color);
+  }
+
+  .content hr {
+    display: block;
+    height: 1px;
+    border: 0;
+    border-top: 1px solid white;
+    margin-top: -1px;
+    padding: 0;
+  }
+</style>
+
+<script>
+  import { selectedResource } from "../stores";
+  import AssemblyView from "./AssemblyView.svelte";
+  import DecompilationView from "./DecompilationView.svelte";
+  import Breadcrumb from "../utils/Breadcrumb.svelte";
+  import HexView from "../hex/HexView.svelte";
+  import TextView from "./TextView.svelte";
+  import { onMount } from "svelte";
+  import Tabs from "../utils/Tabs.svelte";
+  export let resources;
+  let hasTextView = false;
+  let hasAsmView = false;
+  let hasDecompView = false;
+  let tabs = [];
+
+  onMount(async () => {
+    document.getElementById("hex").focus();
```

Comment:
I'm honestly pretty sketched out about using `getElementById` anywhere in Svelte code. I wonder if there is a better way to do this where we pass around actual component instances instead of IDs?

My concern is that this code can act on elements across components in unexpected ways, makes components a little less portable and a little more error-prone, and is also harder for the Svelte compiler to optimize.